src/createAction.js

```diff
@@ -1,22 +1,27 @@
 import identity from 'lodash/identity';
 import isFunction from 'lodash/isFunction';
 import isUndefined from 'lodash/isUndefined';
+import isNull from 'lodash/isNull';
```

Comment:
`undefined` is coerced at the parameter level, so I don't think we need to check for it downstream.